### PR TITLE
Knative-example app assets and documentation

### PR DESCRIPTION
**Pull request by v-ctiutiu**
## Overview

Added Knative serving example app and assets to complement the [DOKS-CI-CD](https://github.com/digitalocean/container-blueprints/tree/main/DOKS-CI-CD) blueprint.

### CODE REVIEW

File: knative-example/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+## Building the 2048 Game Application
+
+Main project is [npm](https://www.npmjs.com) based, hence you can build the application via:
+
+```shell
+npm install --include=dev
+
+npm build
+```
```
Comment (v-bpastiu) [inline knative-example/README.md:15]: Npm build fails for me using npm version 8.9.0.
This should be changed to `npm run build`

File: knative-example/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+First, you need a [DOKS](https://docs.digitalocean.com/products/kubernetes/quickstart) cluster configured and running. Then, the [Knative 1-Click App](https://marketplace.digitalocean.com/apps/knative) must be provisioned to your DOKS cluster.
+
+Next, edit the [knative-service](resources/knative-service.yaml) manifest provided in this repository, and replace the `<>` placeholders accordingly. Finally, apply changes using `kubectl`:
+
+```shell
+kubectl apply -f knative-example/resoures/knative-service.yaml
+```
+
+You can check Knative service status via [kn] CLI:
+
```
Comment (v-bpastiu) [inline knative-example/README.md:70]: This should probably be a link to KN cli.

File: knative-example/README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+The sample manifest provided in this repository will get you started with [Knative Serving](https://knative.dev/docs/serving).
+
+First, you need a [DOKS](https://docs.digitalocean.com/products/kubernetes/quickstart) cluster configured and running. Then, the [Knative 1-Click App](https://marketplace.digitalocean.com/apps/knative) must be provisioned to your DOKS cluster.
+
+Next, edit the [knative-service](resources/knative-service.yaml) manifest provided in this repository, and replace the `<>` placeholders accordingly. Finally, apply changes using `kubectl`:
+
+```shell
+kubectl apply -f knative-example/resoures/knative-service.yaml
+```
+
```
Comment (v-bpastiu) [inline knative-example/README.md:68]: This should probably be a link to KN CLI

File: knative-example/README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+## Deploying to Kubernetes
+
+The sample manifest provided in this repository will get you started with [Knative Serving](https://knative.dev/docs/serving).
+
+First, you need a [DOKS](https://docs.digitalocean.com/products/kubernetes/quickstart) cluster configured and running. Then, the [Knative 1-Click App](https://marketplace.digitalocean.com/apps/knative) must be provisioned to your DOKS cluster.
+
+Next, edit the [knative-service](resources/knative-service.yaml) manifest provided in this repository, and replace the `<>` placeholders accordingly. Finally, apply changes using `kubectl`:
+
+```shell
```
Comment (v-bpastiu) [inline knative-example/README.md:65]: The user was asked a few lines above to change to the knative-example directory. This should be changed to:
kubectl apply -f resources/knative-service.yaml

File: knative-example/README.md (hunk 1 above)
Comment (v-ctiutiu) [reply knative-example/README.md:15]: Thanks. Fixed in latest commit.

File: knative-example/README.md (hunk 3 above)
Comment (v-ctiutiu) [reply knative-example/README.md:68]: Thanks. Fixed in latest commit.

File: knative-example/README.md (hunk 4 above)
Comment (v-ctiutiu) [reply knative-example/README.md:65]: Thanks. Fixed in latest commit.

**Review by v-bpastiu — APPROVED** [review]
Legit

**Review by bikram20 — APPROVED** [review]

**Pull request merged**